compiler-core/src/format/tests.rs

```diff
@@ -6512,3 +6512,32 @@ fn echo_with_following_value_in_a_pipeline() {
 "#
     );
 }
+
+#[test]
+fn echo_printing_multiline_pipeline() {
+    assert_format_rewrite!(
```

Comment:
This doesn't match how any of the other keywords are formatted today. The others use this:

```gleam
echo first
  |> wibble
  |> wobble
```

Reply:
Ok I can go for that as well! Do you think it would be clear that it's printing the whole thing and not just `first` though?
I was afraid people would parse it as `{ echo first } |> wibble |> wobble` instead of `echo { first |> wibble |> wobble }`

Reply:
I think the formatting shifting the indentation by one step will be clear enough 👍

Reply:
Ok!